yarn.lock

```diff
@@ -1787,6 +1787,74 @@
     "@patternfly/react-core" "6.0.0-alpha.94"
     commander "^12.0.0"
 
+"@patternfly/documentation-framework@6.0.0-alpha.94":
```

Comment:
I think these lines in the yarn.lock file should be unnecessary. are they being pulled in if you run `yarn install` again?
Quickstarts shouldn't be requiring you to pull it in, I don't think.

Reply:
Good catch.  Thanks @nicolethoen